src/main/java/com/ericsson/eiffel/remrem/publish/helper/RMQHelper.java

```diff
@@ -61,16 +70,31 @@ public void setExchangeName(String exchangeName) {
         try {
             ConnectionFactory factory = new ConnectionFactory();
             factory.setHost(host);  
-            
+
             if (port != null) {
             	factory.setPort(port);
             	log.info("Port is: " + port);
             } else {
             	log.info("Using default rabbit mq port.");
             }
-            //factory.useSslProtocol();
+            
             log.info("Host adress: " + host);
             log.info("Exchange is: " + exchangeName);
+            // "TLSv1.2"
+            if (tls_ver != null && !tls_ver.isEmpty()) {
+            	if (tls_ver.contains("default")) {
+            		log.info("Using default TLS version connection to RabbitMq.");
+            		factory.useSslProtocol();
+            	}
+            	else {
+               		log.info("Using TLS version " + tls_ver + " connection to RabbitMq.");
```

Comment:
Indentation is off. While fixing perhaps replacing all tabs with spaces in this file would prevent this from happening again.